modules/ml/src/main/java/org/apache/ignite/ml/dataset/impl/local/LocalDatasetBuilder.java

```diff
@@ -127,16 +168,24 @@ public LocalDatasetBuilder(Map<K, V> upstreamMap, IgniteBiPredicate<K, V> filter
      * @param <T> Target type of entries.
      */
     private static class IteratorWindow<K, T> implements Iterator<T> {
-        /** Delegate iterator. */
+        /**
```

Comment:
Please walk through the code and remove such changes. They are not required, but make this PR harder to review.